packages/console-sdk-microfrontend/src/MicrofrontendIntegrator.ts

```diff
@@ -38,18 +38,18 @@ export default class MicrofrontendIntegrator implements IMicrofrontendIntegrator
   private configObservable: ISDKConsoleObservable
   private consoleWriteConfig: IWriteConfig
 
-  constructor(mountingProps: IConsoleProps) {
-    const { name, console, container } = getConsoleProps(mountingProps)
-    const { eventBus, contexts } = console
+  constructor(mountingProps?: IConsoleProps) {
+    const consoleProps = mountingProps ? getConsoleProps(mountingProps) : {} as ISDKProps
```

Comment:
I'd deal with the defaults within the getConsoleProps function:

https://github.com/mia-platform/console-sdk/blob/3c84def2af76aaa37a7dbb0e5cd35460cf2b6819/packages/console-sdk-microfrontend/src/adaptConsoleProps.ts#L24

using 

```
export function getConsoleProps(props: IConsoleProps = {}): ISDKProps {
```

and managing proper default props that could still be handled by the SDK

Reply:
okay, done. what do you think so?